Makefile

```diff
@@ -73,6 +73,10 @@ prestate: build-rvgo op-program-riscv
 	mv ./rvgo/bin/0.json ./rvgo/bin/prestate-proof.json
 .PHONY: prestate
 
+cannon-prestate:
```

Comment:
What still needs cannon here?

Reply:
I added this check because upstream README asked to build cannon-prestate to generate devnet allocs: https://github.com/ethereum-optimism/optimism/tree/develop/op-e2e#op-e2e

Reply:
Will double check if this is a redundancy and should be removed.